mantidproject/mantid
# Workbench default layout

* Added the weighting fraction for height and width of columns/rows in the workbench.
* Set up the default layout to be correct upon launch
* Ensured that the max width/height is taken from full screen area upon launch (previously relied on defaults in `CONF`)
* Added a `View > Restore Default Layout` button in the QMenu which reverts the current state of the interface to what you see on launch.  

**To test:**

* Run the workbench and check the layout is as expected (described in the issue)
* Check you can drag the vertical and horizontal boundaries of each widget and all widget movement is as expected.
* Delete/Move/Reorder widget and the press the `View > Restore Default Layout` button in the menu. (Do the same but with `Shift+F10`: shortcut for the button)   

Fixes #21566

**Release Notes** 
*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
